topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.md

```diff
@@ -485,4 +516,46 @@ The next example is *SRR12416842* from an Indonesia [study](https://www.microbio
 
 There is something clearly wrong with sample SRR12416842, perhaps indicating sample contamination. This example of a sample that doesn't map to the reference genome illustrates that even when sequence quality is good, sequence data problems can become apparent in later steps of analysis and it is important to always have a sense of what results to expect. You can develop a better sense of what quality control results to expect by first practicing techniques with known data before analysing new samples.
 
-We hope you enjoyed this tutorial!
+# Processing many samples at once: Collections and Workflows (optional)
+
+In the tutorial thus far we have focused on processing single samples, where two read datasets (forward and reverse reads) are associated with a single sample. In practice, sequence analysis typically involves analysing batches of samples and we run the same
+analysis steps for each sample in the batch. Galaxy supports working with batches using collections and workflows. 
+
+If you are new to collections, you can learn more about them in [this tutorial]({% link topics/galaxy-interface/tutorials/collections/tutorial.md %}).
+
+If you have followed all of the steps of this tutorial, you will have two samples, one named *004-2* and another *SRR12416842*. Create a collection of lists of pairs from these samples and call the collection `samples`.
```

Comment:
> a collection of lists of pairs

I guess you mean just a "List of Dataset Pairs"? Collection of lists of pairs implies more nesting to me.
Either way, instructions how to do this could be nice without having people go through the collections tutorial.

Reply:
I have changed the wording as provided and include a snippet on making a list of pairs instead of directing people to the tutorial.